pkg/governance-scripts/contracts/deprecated/20220704-timelock-authorizer-migrator/README.md

```diff
@@ -0,0 +1,12 @@
+# Timelock Authorizer Migrator
```

Comment:
As this was never executed I think we can just delete this and the deployment task rather than deprecating it.